cammac60/MacRae-Photo
# Testing

### What does this PR do? 

- Adds tests for all components.
- Fixes failing apiCall test.
- Removes unneeded lines/minor refactoring.

----------

### How to manually test: 

- `git fetch`
- `git checkout testing`
- `npm test` to ensure all tests are passing. 

----------

### Tickets: 

- Resolves #42 
- Resolves #18 

----------

### Notes: 

- None.
